a/github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml b/github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml
index 3483e309..79798046 100644
--- a/github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml
+++ b/github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml
@@ -3,7 +3,7 @@ postsubmits:
   - name: push-release-containerized-data-importer-images
     branches:
     - release-v\d+\.\d+
-    cluster: kubevirt-prow-control-plane
+    cluster: kubevirt-prow-workloads
     always_run: true
     optional: false
     skip_report: true
@@ -11,13 +11,15 @@ postsubmits:
       testgrid-create-test-group: "false"
     decorate: true
     decoration_config:
-      timeout: 1h
+      timeout: 2h
       grace_period: 5m
     labels:
       preset-podman-in-container-enabled: "true"
       preset-docker-mirror-proxy: "false"
       preset-kubevirtci-quay-credential: "true"
     spec:
+      nodeSelector:
+        type: bare-metal-external  
       containers:
       - image: quay.io/kubevirtci/bootstrap:v20231115-51a244f
         env:
@@ -29,9 +31,10 @@ postsubmits:
         - "-c"
         - |
           cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io &&
-          # Only push images on tags
+          # Only push images on tags, for amd64 and aarch64 (crossbuild, since we have no aarch64 builder)
           [ -z "$(git tag --points-at HEAD | head -1)" ] ||
-          DOCKER_TAG="$(git tag --points-at HEAD | head -1)" make bazel-push-images
+          DOCKER_TAG="$(git tag --points-at HEAD | head -1)-amd64" make bazel-push-images ||
+          DOCKER_TAG="$(git tag --points-at HEAD | head -1)-arm64" BUILD_ARCH="crossbuild-aarch64" make bazel-push-images
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
